Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -80,11 +83,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
     }
 
-    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-        storage.deviceToken = String(apnDeviceToken: deviceToken)
-        MessagingPush.shared.application(application, didRegisterForRemoteNotificationsWithDeviceToken: deviceToken)
-    }
-
     func application(_ application: UIApplication, didFailToRegisterForRemoteNotificationsWithError error: Error) {
         MessagingPush.shared.application(application, didFailToRegisterForRemoteNotificationsWithError: error)
     }
```

Comment:
Can we remove this function, too? Or also swizzle it?

Reply:
Swizzle.d